utils/parametric/spec/otel_trace.py

```diff
@@ -62,3 +52,61 @@ def convert_to_proto(attributes: dict) -> pb.Attributes:
         if ret:
             list_attr[k] = ret
     return pb.Attributes(key_vals=list_attr)
+
+
+class SpanKind(Enum):
```

Comment:
This enum should be in `utils/dd_constants.py` ? 

It would be a good idea to also add a link to the RFC.

Same comment for enum above

Reply:
This is this a big refactor. I moved these changes to their own PR: https://github.com/DataDog/system-tests/pull/3370/files